frontend/src/bundles/common/components/sort/sort.tsx

```diff
@@ -0,0 +1,55 @@
+import clsx from 'clsx';
+import {
+    type ButtonHTMLAttributes,
+    type MouseEventHandler,
+    type ReactNode,
+} from 'react';
+
+import { IconName } from '../../enums/enums';
+import { Icon } from '../components';
+import styles from './styles.module.scss';
+
+type SortType = 'asc' | 'desc' | null;
+
+interface Properties extends ButtonHTMLAttributes<HTMLButtonElement> {
+    sortType?: SortType;
+    iconFontSize?: string;
+    defaultIconColor?: string;
```

Comment:
From the outside of the component I would like to see a list of available colors; this will not work with the string type.
And this can also be passed through a class, so this property is redundant